README.md

```diff
@@ -192,16 +193,84 @@ metrics = require('metrics')
 #### `histogram_obj:observe(num, label_pairs)`
   Records a new value in histogram. This increments all buckets sizes under labels `le` >= `num` and labels matching `label_pairs`.
   * `num` Value to put in histogram (number).
-  * `label_pairs` Table containing label names as keys, label values as values (table). New value is observed by all internal counters with these labels specified 
+  * `label_pairs` Table containing label names as keys, label values as values (table). New value is observed by all internal counters with these labels specified.
 
 #### `histogram_obj:collect()`
   Returns concatenation of `counter_obj:collect()` across all internal counters
   of `histogram_obj`.  
   For `observation` description see `counter_obj:collect()` section.
 
+#### Average
+  Can be used only as HTTP statistics collector (described below) and cannot be built explicitly.
+
+#### `average_obj:collect()`
+  Returns list of two observations: 
+  * `name .. "_avg"` - average value of observations for the observing period (time from previous collect call to now),
+  * `name .. "_count"` - observation count for the same period.
+  For `observation` description see `counter_obj:collect()` section.
+
+### Collecting HTTP requests latency statistics
+
+`metrics` also provides a middleware for monitoring HTTP (set by [http](https://github.com/tarantool/http) module) latency statistics.
+
+```
+-- importing submodule
+http_middleware = metrics.http_middleware
+```
+
+#### `http_middleware.configure_default_collector(type_name, name, help)`
+  Registers collector for middleware and sets it as default.
+  * `type_name` Collector type (string): "histogram" or "average". Default is "histogram".
+  * `name` Collector name (string). Default is "http_server_request_latency".
+  * `help` (optional) Help description (string). Default is "HTTP Server Request Latency".
+
+  If collector with the same type and name already exists in registry, throws an error.
```

Comment:
I think this is not worst to mention, and it makes doc more complicated.

However you can keep it if you want.

Reply:
let's keep it for now